Makefile

```diff
@@ -1,8 +1,10 @@
 .PHONY: build generate-completion test test-cover view-cover
 
 TAG=$(shell git describe --tags 2> /dev/null || echo unknown)
+GIT_COMMIT=$(shell git rev-parse HEAD 2> /dev/null || echo unknown)
+CLC_VERSION=$(shell git tag | sort -V | tail -1 2> /dev/null || echo unknown)
```

Comment:
Could you change `CLC_VERSION` to:
```
git describe --tags $(git rev-list --tags --max-count=1) || echo UNKNOWN
```

Reply:
But this command gives a hash value, not the latest release tag. 

For example: `b11c5bdef4b055b2a28c1d1895b8d07ae990d2ec-1-g33465e9`

Reply:
It outputs this for me:
```
$ git describe --tags $(git rev-list --tags --max-count=1) || echo UNKNOWN
v1.0.0-beta1
```
Maybe you didn't fetch the tags?